recipe/recipe.yaml

```diff
@@ -12,15 +12,19 @@ source:
   patches: remove-getopt-required.patch
 
 build:
-  number: 1
+  number: 2
   script:
     # Get an updated config.sub and config.guess
-    - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+    - if: unix
+      then: cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -S . -B build
+    - if: win
+      then: cmake %CMAKE_ARGS% -S . -B build
     - cmake -LH build
-    - cmake --build build --parallel="${CPU_COUNT}"
+    - if: unix
+      then: cmake --build build --parallel="${CPU_COUNT}"
+    - if: win
+      then: cmake --build build --config=Release --parallel="%CPU_COUNT%"
```

Comment:
Should `--config=Release` get moved up into the `cmake` configuration command instead of passing it at build time?

Reply:
No - as far as I can tell messing around on my desktop - this is a particular oddity of MSVC that you have to specify this at build time rather than configure time. I am happy to entertain suggestions of folks with more experience.

Reply:
If we change to using Ninja then we can define release up at configure time. I don't really see any major benefit or drawbacks either way, to be honest.